app/controllers/consultations_controller.rb

```diff
@@ -69,6 +79,7 @@ def set_consultation
 
     # Never trust parameters from the scary internet, only allow the white list through.
     def consultation_params
-      params.fetch(:consultation, {})
```

Comment:
Why .fetch rather than `params.permit(:consultations).require(:user_id, :legislation_id)`  ?

Reply:
that was just the standard that was there when I scaffolded the consultation controller - of course tried to normal way ran into issue so then just left it because was testing for navigation and flow initially